webui/react/src/hooks/useModal/UserSettings/useModalCreateUser.tsx

```diff
@@ -112,69 +85,33 @@ const ModalForm: React.FC<Props> = ({ form, user, groups, viewOnly, roles }) =>
       <Form.Item label={DISPLAY_NAME_LABEL} name={DISPLAY_NAME_NAME}>
         <Input disabled={viewOnly} maxLength={128} placeholder="Display Name" />
       </Form.Item>
-      <Form.Item label="Configure Agent" name="useAgent" valuePropName="checked">
-        <Switch disabled={viewOnly} />
-      </Form.Item>
-      {useAgent && (
-        <>
-          <Form.Item
-            label="Agent User ID"
-            name="agentUid"
-            rules={[{ message: 'Agent User ID is required ', required: true }]}>
-            <InputNumber disabled={viewOnly} />
-          </Form.Item>
-          <Form.Item
-            label="Agent User Name"
-            name="agentUser"
-            rules={[{ message: 'Agent User Name is required ', required: true }]}>
-            <Input disabled={viewOnly} maxLength={100} />
-          </Form.Item>
-          <Form.Item
-            label="Agent User Group ID"
-            name="agentGid"
-            rules={[{ message: 'Agent User Group ID is required ', required: true }]}>
-            <InputNumber disabled={viewOnly} />
-          </Form.Item>
-          <Form.Item
-            label="Agent Group Name"
-            name="agentGroup"
-            rules={[{ message: 'Agent Group Name is required ', required: true }]}>
-            <Input disabled={viewOnly} maxLength={100} />
-          </Form.Item>
-        </>
-      )}
       {!rbacEnabled && (
         <Form.Item label={ADMIN_LABEL} name={ADMIN_NAME} valuePropName="checked">
           <Switch disabled={viewOnly} />
         </Form.Item>
       )}
-      {!user && rbacEnabled && (
-        <Form.Item label={GROUP_LABEL} name={GROUP_NAME}>
-          <Select
-            mode="multiple"
-            optionFilterProp="children"
-            placeholder="Select Groups"
-            showSearch>
-            {groups.map((u) => (
-              <Select.Option key={u.group.groupId} value={u.group.groupId}>
-                {u.group.name}
-              </Select.Option>
-            ))}
-          </Select>
-        </Form.Item>
-      )}
       {rbacEnabled && canModifyPermissions && (
         <>
-          <Form.Item label={ROLE_LABEL} name={ROLE_NAME}>
+          <Form.Item
+            label={
+              <span>
+                {ROLE_LABEL}{' '}
+                <Icon
+                  name="info"
+                  size="small"
+                  title="This user role will be applied for all workspaces and areas across the cluster."
+                />
+              </span>
+            }
+            name={ROLE_NAME}>
             <Select
               disabled={(user !== undefined && roles === null) || viewOnly}
-              mode="multiple"
```

Comment:
I wonder why are we removing multiple for global roles? Also if we change staff here, we need to modify how we call API

Reply:
discussed with @alexjdetermined, removing this from the ticket.